sharktank/tests/models/t5/t5_test.py

```diff
@@ -501,7 +498,11 @@ def testCompareToyIreeVsEager(
         reference_dtype: torch.dtype,
         atol: float,
         rtol: float,
+        xfail_reason: str | None = None,
     ):
+        if xfail_reason is not None:
+            pytest.xfail(reason=xfail_reason)
+
```

Comment:
No need for this if using the aforementioned pattern.
https://stackoverflow.com/a/30575822/324204